src/containers/Tenant/Diagnostics/TenantOverview/Healthcheck/HealthcheckPreview.tsx

```diff
@@ -45,17 +63,14 @@ export function HealthcheckPreview(props: HealthcheckPreviewProps) {
                         onClick={(event) => {
                             // FIXME: refactor card to remove the button from the anchor link.
                             event.preventDefault();
-                            onUpdate();
+                            refetch();
```

Comment:
If we refetch overview, but details are opened, it seems we should refetch them either.

Reply:
Discussed on standup, that when we open healthcheck details, we need also update healthcheck overview, so the data will be consistent.